docs/get-started.md

```diff
@@ -11,13 +11,38 @@ weight=-70
 
 # Get started with Docker Machine and a local VM
 
-Let's take a look at using `docker-machine` for creating, using, and managing a
-Docker host inside of <a href=
-"https://www.virtualbox.org/" target="_blank">VirtualBox</a>.
+Let's take a look at using `docker-machine` to create, use and manage a
+Docker host inside of a local virtual machine.
+
+## Prerequisite Information
+
+With the advent of [Docker for Mac](/docker-for-mac/index.md) and [Docker for Windows](/docker-for-windows/index.md) as replacements for [Docker Toolbox](/toolbox/overview.md), there are some new considerations for creating local machines with `docker-machine create`.
+
+The new solutions come with their own native virtualization solutions rather than using Oracle VirtualBox. In the case of Docker for Windows, you'll use a different driver to create machines.
+
+Like Toolbox, both Docker for Mac and Docker for Windows automatically include the newest version of Docker Machine, so when you install either of these, you still get `docker-machine` on your system.
+
+#### If you are using Docker for Windows
+
+Docker for Windows uses [Microsoft Hyper-V](https://msdn.microsoft.com/en-us/virtualization/hyperv_on_windows/windows_welcome) for virtualization, and Hyper-V and Oracle VirtualBox are incompatible. You cannot run the two solutions simultaneously. But you can still use `docker-machine` to create local VMs by using the Microsoft Hyper-V driver.
```

Comment:
Yeah +1 I think if possible to combine this message with the above sentiment I commented on  would be good